04 Abstraction Exercises Part 1/hotel-room-application/src/main/java/com/hotel/room/application/HotelRoom.java

```diff
@@ -0,0 +1,50 @@
+package com.hotel.room.application;
+
+/**
+ * This is the Hotel Room Class
+ * */
+
+public class HotelRoom {
+
+    /**
+     * This part are the Fields
+     * */
+
+    private int NumOfAdults;
```

Comment:
simplify the fields. since there are no difference between the charges per person of adult and children, you might want to use one field for both

Reply:
updated changes na po sir